package.json

```diff
@@ -150,5 +150,9 @@
     "colors": "1.4.0",
     "@types/prettier": "2.6.0"
   },
-  "//": "~~ Generated by projen. To modify, edit .projenrc.js and run \"npx projen\"."
-}
\ No newline at end of file
+  "//": "~~ Generated by projen. To modify, edit .projenrc.js and run \"npx projen\".",
+  "bundleDependencies": [
```

Comment:
this file is generated by [projen][https://github.com/projen/projen), any change should be made in `.projenrc.js`.